include/simdjson/padded_string.h

```diff
@@ -24,7 +35,9 @@ inline char *allocate_padded_buffer(size_t length) noexcept {
     return nullptr;
   }
 #endif // NDEBUG
-  memset(padded_buffer + length, 0, totalpaddedlength - length);
+  
+    // ? -> memset(padded_buffer + length, 0, totalpaddedlength - length);
```

Comment:
Why would you remove the memset?

Reply:
if not strictly required that will improve the performance. What do you think? Although on -O3 that `memset` should be instant. Perhaps.  I am not sure. Testing as we speak.

Reply:
Have you determined that it is not required?

Reply:
Not sure ... here hitting head one differences between runtimes and compilers.  Leaving it in.